src/shared/Navbar/index.tsx

```diff
@@ -1,19 +1,34 @@
 import AppBar from '@material-ui/core/AppBar';
+import { makeStyles } from '@material-ui/core/styles'; 
 import Typography from '@material-ui/core/Typography';
 import Toolbar from '@material-ui/core/Toolbar';
 import React from 'react';
+import CartButton from '../CartButton'
 
-const withNavbar = (Component: React.FC): React.FC => () => (
+const withNavbar = (Component: React.FC): React.FC => () => {
+  const classes = styles();
+  return(
   <>
     <AppBar position="sticky" color="primary">
       <Toolbar>
         <Typography variant="h6" color="inherit">
           GeekText
         </Typography>
+        {/* Place any buttons that need to be right-aligned in this div. */}
```

Comment:
Maybe we could split this off into a component called RightAlignedButtons? I think it's a pretty good rule of thumb to think about splitting something off into its own component if you feel like you should write a comment like this.